src/components/account/login.vue

```diff
@@ -69,10 +79,60 @@ export default {
       password: ''
     };
   },
+  computed: {
+    oidcLoginPath() {
+      const { query } = this.$route;
+      const next = typeof query.next === 'string' ? query.next : null;
+      const qs = queryString({ next });
+      return `/v1/oidc/login${qs}`;
+    }
+  },
+  created() {
+    const { oidcError } = this.$route.query;
+    if (this.config.oidcEnabled && typeof oidcError === 'string' &&
+      /^[\w-]+$/.test(oidcError)) {
```

Comment:
I guess regex could be more precise if wanted, e.g. `/^[a-z-]+$`.

Reply:
Who knows, maybe we'll have an error message with a number one day. 🤣 In some ways, the primary check is the next one, that there is an i18n message associated with the value of `oidcError`. To complete that check, we call `$te()`, but there are some values that I think would cause issues with `$te()`, for example, `..`. So this regular expression is here to check, "is this value well-formed enough to pass to `$te()`?"